astrology.lic

```diff
@@ -276,15 +285,30 @@ class Astrology
     fix_standing
   end
 
-  def train_astrology
-    check_ripples
-    check_astral
-    check_weather
-    check_events(check_pools) if @predict_event
-    check_moons
-    check_heavens
+  def train_astrology(settings)
+    return unless settings
+    return unless settings.astrology_training
+
```

Comment:
Since this is a new setting, do you want to explicitly check for the case in which `settings.astrology_training` is `[]` and output a message (e.g. 'Go set settings.astrology_training - see base.yaml for an example')?